### ISSUE TITLE
Standard does not work in PHPStorm

### ISSUE THREAD

**Issue by androidacy-user** [standard/standard#1835]
**What version of this package are you using?** Trying to use v17

**What operating system, Node.js, and npm version?** Windows 11, Node v18.4.0, NPM v8.17.0

**What happened?** v17 Throws an error when trying to use it with PHPStorms linting. v16 works fine.

```
Initialization error (ESLint). require() of ES Module .\api-next\node_modules\standard\index.js from .\AppData\Local\JetBrains\Toolbox\apps\PhpStorm\ch-0\222.3345.135\plugins\JavaScriptLanguage\languageService\eslint\bin\eslint-common.js not supported.
    Instead change the require of index.js in .\AppData\Local\JetBrains\Toolbox\apps\PhpStorm\ch-0\222.3345.135\plugins\JavaScriptLanguage\languageService\eslint\bin\eslint-common.js to a dynamic import() which is available in all CommonJS modules.
    Error [ERR_REQUIRE_ESM]: require() of ES Module .\api-next\node_modules\standard\index.js from .\AppData\Local\JetBrains\Toolbox\apps\PhpStorm\ch-0\222.3345.135\plugins\JavaScriptLanguage\languageService\eslint\bin\eslint-common.js not supported.
    Instead change the require of index.js in .\AppData\Local\JetBrains\Toolbox\apps\PhpStorm\ch-0\222.3345.135\plugins\JavaScriptLanguage\languageService\eslint\bin\eslint-common.js to a dynamic import() which is available in all CommonJS modules.
    at requireInContext (.\AppData\Local\JetBrains\Toolbox\apps\PhpStorm\ch-0\222.3345.135\plugins\JavaScriptLanguage\languageService\eslint\bin\eslint-common.js:24:12)
at new ESLintPlugin (.\AppData\Local\JetBrains\Toolbox\apps\PhpStorm\ch-0\222.3345.135\plugins\JavaScriptLanguage\languageService\eslint\bin\eslint-plugin.js:24:72)
at ESLintPluginFactory.create (.\AppData\Local\JetBrains\Toolbox\apps\PhpStorm\ch-0\222.3345.135\plugins\JavaScriptLanguage\languageService\eslint\bin\eslint-plugin-provider.js:17:34)
at Interface.<anonymous> (.\AppData\Local\JetBrains\Toolbox\apps\PhpStorm\ch-0\222.3345.135\plugins\JavaScriptLanguage\jsLanguageServicesImpl\js-language-service.js:82:52)
at Interface.emit (node:events:537:28)
at Socket.emit (node:events:537:28)
Process finished with exit code -1
```

**What did you expect to happen?** Work correctly

**Are you willing to submit a pull request to fix this bug?** Don't even know where to start

**Comment by oneserv-heuser** [comment]
Can confirm this issue with Windows 10, Node v16.17.0, npm 8.15.0. v16 works and the update to v17 breaks it.

But I think that this issue has to be fixed by Jetbrains (publisher of PHPStorm) and not standard.

**Comment by oneserv-heuser** [comment]
The issue seems to be fixed now with the latest PHPStorm version 2022.3.2.

**Comment by LinusU** [comment]
That's great, thanks for reporting back!
